modules/tensor_mechanics/doc/content/modules/tensor_mechanics/NewMaterialSystem.md

```diff
@@ -0,0 +1,135 @@
+| Deformation gradient                     | $F_{iJ} = \delta_{iJ} + \frac{\partial u_i}{\partial X_J}$                    | $F_{ij} = \delta_{ij} + \frac{\partial u_i}{\partial x_j}$ |
+| Inverse deformation gradient             | $F^{-1}_{Ji}$                                                                 | $\delta_{ji}$                                              |
+| Inverse incremental deformation gradient | $f^{-1}_{ij} = F^{(old)}_{iK} F^{(new) -1}_{Kj}$                              | $f_{ij} = \delta_{ij}$                                     |
+| Volume change                            | $J = \det F$                                                                  | $J = 1$                                                    |
+| Total strain increment                   | $\Delta d_{ij} = \frac{1}{2} \left( l_{ij} + l_{ji} \right)$                  | Same                                                       |
+| Mechanical strain increment              | $\Delta \varepsilon_{ij} = \Delta d_{ij} - \Delta \varepsilon_{ij}^{(eigen)}$ | Same                                                       |
+| Total strain                             | $d_{ij}^{(new)} = d_{ij}^{(old)} + \Delta d_{ij}$                             | Same                                                       |
+| Mechanical strain                        | $\varepsilon_{ij} = \varepsilon_{ij}^{(old)} + \Delta \varepsilon_{ij}$       | Same                                                       | 
+
+Except for the deformation gradient itself, all the large deformation quantities used to map configurations are set to the identity 
+for `large_kinematics = false`, leaving only the strain tensors defined.
+
+The strains are defined incrementally in terms of the inverse incremental deformation gradient (or the increment in the gradient for small displacement kinematics).
+The calculator [stabilizes](/tensor_mechanics/Stabilization.md) the deformation gradient adds any homogenization gradient from the [homogenization system](tensor_mechanics/Homogenization.md)
+before calculating the incremental and accumualted strains, and so these modifications propogate through all the available kinematic measures.
+However, the eigenstrains are currently defined incrementally and so they will not affect the deformation gradient, the inverse deformation gradient,
+and the incremental deformation gradient.
+
+## The `large_kinematics` Option
+
+The [`ComputeLagrangianStrain`](ComputeLagrangianStrain.md) class uses the `large_kinematics` option to switch between
+large strain kinematics and small strain kinematics.  This option must be set consistently between the strain 
+calculator and the kernels.  The [TensorMechanics/MasterAction](/Modules/TensorMechanics/Master/index.md) can
+be used to automatically setup the kernels and strain calculator with consistent options.
```

Comment:
Probably worth saying that you use the `strain=FINITE` option in that action to get this.

Reply:
Added a mention.